recipes/fmt/all/conanfile.py

```diff
@@ -13,8 +15,20 @@ class FmtConan(ConanFile):
     exports_sources = ["CMakeLists.txt", "patches/**"]
     generators = "cmake"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "header_only": [True, False], "fPIC": [True, False], "with_fmt_alias": [True, False]}
-    default_options = {"shared": False, "header_only": False, "fPIC": True, "with_fmt_alias": False}
+    options = {
+        "header_only": [True, False],
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_fmt_alias": [True, False],
+        "with_os": [True, False],
```

Comment:
I don't really like the `with_os` name. It's too close to `self.settings.os`
See https://github.com/fmtlib/fmt/issues/1654#issuecomment-621218600
How about `with_os_api` or something similar?

Reply:
That's a good point. I second `with_os_api`.

Reply:
done